src/options/options.js

```diff
@@ -20,11 +21,19 @@ const buildForm = async () => {
         if (folder.type !== "trash") {
           await addFormFolder(accountsListEl, account, folder);
         }
-      } else {
+      } else if (folder.type === "inbox") {
         // Only the "inbox" of the mail account can be selected.
```

Comment:
Please remove this comment. Otherwise everything looks good to me. I'll merge and update the doc.

Reply:
Done.